modules/http/src/main/java/org/glassfish/grizzly/http/HttpServerFilter.java

```diff
@@ -561,7 +561,11 @@ private void prepareRequest(final ServerHttpRequestImpl request, final boolean h
         }
 
         final MimeHeaders headers = request.getHeaders();
-
+        //here we can add validation to prevent Http Server Smuggling
+        if (headers.contains(Header.ContentLength.getLowerCase()) && headers.contains(Header.TransferEncoding.getLowerCase())) {
```

Comment:
I don't believe we need to use .getLowerCase()

Internally the following is use within the headers.contains method:
```
if (headers[i].getName().equalsIgnoreCase(name)) {
                return i;
            }
```

Reply:
Other than this comment, the PR looks good